rest-api-spec/src/main/resources/rest-api-spec/test/scroll/11_clear.yaml

```diff
@@ -76,3 +76,43 @@
         catch: missing
         clear_scroll:
           scroll_id: $scroll_id1
+
+---
+"Body params with string param scroll id override query string":
```

Comment:
This test fails the backward compatibility check with 5.x. Can you add a skip section like in https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/test/indices.exists_template/10_basic.yaml that will disable the BWC test for this. I'll reenable them when this is back-ported to 5.x.